django_ontruck/notifiers/sms_notifier.py

```diff
@@ -0,0 +1,47 @@
+from abc import abstractmethod
+from .notifier import Notifier
+from .sms_locmem_client import SmsLocMemClient
+from .async_notifier import AsyncNotifier, MetaDelayedNotifier
+
+
+class SMSNotifier(Notifier, metaclass=MetaDelayedNotifier):
+    __slots__ = ('phone',)
+    async_class = AsyncNotifier
+
+    @classmethod
+    def is_default_delayed(cls):
+        return True
+
+    def __new__(cls, *args, **kwargs):
+        is_delayed = kwargs.pop('delayed', False)
+
+        if is_delayed:
+            return cls.async_class(*args, notifier_class=cls, **kwargs)
+        else:
+            return super().__new__(cls)
+
+    def __init__(self, phone):
+        self.phone = phone
```

Comment:
@mmacia Does this `phone` variable refer to the phone number? If so, I would propose to rename the variable to be more explicit, such as `phone_number`.

Reply:
:+1: